packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -976,6 +976,8 @@ errors.E0000151 = 》Ýöû ĥåṽé ŕéåçĥéð ţĥé ļîɱîţ öƒ çå
 oie.invalid.recovery.token = 》Ţĥé ŕéçöṽéŕý ţöķéñ îš îñṽåļîð· 䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.consent.scopes.admin.title = 》ŵöûļð ļîķé ţö åççéšš∶ Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.consent.scopes.enduser.title = 》ŵöûļð ļîķé ţö∶ โ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.consent.scopes.granular.title = 》≤€①≥ ฐโ{0}≤⁄€①≥ ≤€②≥ŕéǫûéšţš åççéšš ţö≤⁄€②≥ Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
Ideally, HTML tags should not be translated. 
It is not related to this PR, but I think it might be caused by pseudo-loc library version outdated.

Reply:
Is it ok to leave this in? Or should I manually fix the tags in this file?

Reply:
It's ok for now, all the other values in this file are like that already.